src/Core/Services/Implementations/OrganizationSponsorshipService.cs

```diff
@@ -37,41 +37,41 @@ public class OrganizationSponsorshipService : IOrganizationSponsorshipService
             _dataProtector = dataProtectionProvider.CreateProtector("OrganizationSponsorshipServiceDataProtector");
         }
 
-        public async Task<bool> ValidateRedemptionTokenAsync(string encryptedToken, string sponsoredUserEmail)
+        public async Task<(bool valid, OrganizationSponsorship sponsorship)> ValidateRedemptionTokenAsync(string encryptedToken, string sponsoredUserEmail)
```

Comment:
Love the use of ValueTuple but do we want convention to be camelCase or PascalCase. I believe the ones I have used so far I went with Pascal.

Reply:
For the names? No they're usually camelCase. The types are Pascal and not all of our ValueTuples are named. Maybe that's the confusion?

Reply:
I've seen Pascal personally and this is someone from Microsoft saying pascal. https://github.com/dotnet/runtime/issues/27939#issuecomment-531420515

Even though they are technically fields I have always treated them as names for a public property with only a getter. When deconstructing them through `var (one, two) = ...` you make it camel but that is shorthand of:
```csharp
var tuple = GetTuple();
var one = tuple.One;
var two = tuple.Two;
```